--- /data/user/work/netune_extracted/Netune-beta/src/compat/dirent.h	2026-07-28 16:23:54.000000000 +0000
+++ /workspace/Netune-beta/src/compat/dirent.h	2026-07-29 09:45:50.611319567 +0000
@@ -10,7 +10,7 @@
  * receives UTF-8 strings; conversion to/from UTF-16 is handled internally.
  */
 
-#ifdef _MSC_VER
+#ifdef _WIN32
 
 #include <stdlib.h>
 #include <string.h>
@@ -120,4 +120,4 @@
     return 0;
 }
 
-#endif /* _MSC_VER */
+#endif /* _WIN32 */
